OWNERS

```diff
@@ -1,20 +1,23 @@
 reviewers:
   - abhat
```

Comment:
remove aniket and zshi from reviewer list - they are only backup approvers if they get auto picked by bot its a waste